docs/generate_docs.py

```diff
@@ -0,0 +1,169 @@
+                text += '\n      - '
+            else:
+                text += '\n\n   * '
+        elif html_element.name == 'strong':
+            prev_tag = 'strong'
+            text += '<b>'
+        elif html_element.name == 'pre':
+            if ' '.join(html_element.attrs['class']) == 'code shell literal-block':
+                text += ' '
+            elif ' '.join(html_element.attrs['class']) == 'code literal-block':
+                text += '\n<dim>\n'
+                prev_tag = 'code'
+        elif html_element.name == 'cite':
+            text += ' '
+            prev_tag = 'cite'
+        elif html_element.name == 'span':
+            if ' '.join(html_element.attrs['class']) == 'green':
+                text += ' ' + AnsiEscapeCodes.GREEN
+                prev_tag = 'color'
+            if ' '.join(html_element.attrs['class']) == 'yellow':
+                text += ' ' + AnsiEscapeCodes.YELLOW
+                prev_tag = 'color'
+            if ' '.join(html_element.attrs['class']) == 'red':
+                text += ' ' + AnsiEscapeCodes.RED
+                prev_tag = 'color'
+            if ' '.join(html_element.attrs['class']) == 'grey':
+                text += ' ' + AnsiEscapeCodes.DIM
+                prev_tag = 'color'
+    return text
+
+
+def skip_or_replace_unparseable_directives(rst: str) -> str:
+    rst = rst.replace(':ref:', '')\
+             .replace('.. code-block:: ini', '.. code-block::')
+    return rst
+
+
+def resolve_includes(rst: str, docs_source_path: str) -> str:
+    matches = re.findall(r'\.\. include:: (.*)', rst)
+    for match in matches:
+        with open(f'{docs_source_path}/{match}', 'r') as handle:
+            include_text = handle.read()
+        rst = rst.replace(f'.. include:: {match}', include_text)
+    return rst
+
+
+def skip_prefix_new_lines(txt: str) -> str:
+    txt = re.sub(r'\A[\n]+', '', txt)
+    return txt
+
+
+if __name__ == '__main__':
+    output_docs_path = 'git_machete/long_docs.py'
+    docs_source_path = 'docs/source'
+    output_text = 'from typing import Dict\n\nlong_docs: Dict[str, str] = {\n'
+    path = docs_source_path + '/cli_help'
+    commands_and_file_paths = {f.split('.')[0]: join(path, f) for f in sorted(listdir(path)) if isfile(join(path, f))}
+
+    # cmd = 'traverse'
+    # commands_and_file_paths = {cmd: f'source/cli_help/{cmd}.rst'}
+    for command, file in commands_and_file_paths.items():
+        with open(file, 'r') as f:
+            rst = f.read()
+
+        rst = skip_or_replace_unparseable_directives(rst)
+        rst = resolve_includes(rst=rst, docs_source_path=docs_source_path)
+        html = rst2html(rst)['body']
+        # print(html)
+        # print('\n\n\n)
+        plain_text = html2txt(html)
+        plain_text = skip_prefix_new_lines(plain_text)
+        # print(plain_text)
+        output_text += f'    "{command}": """\n' + indent(plain_text, '        ') + '\n   """,\n'
```

Comment:
Maybe even just build a dict here, and add its `repr` into `output_text`? instead of appending pieces of Python code manually

Reply:
@PawelLipski I tried but there is no easy way to write the dict as string in the readable format so i would stick with how it is atm